resource/repository.go

```diff
@@ -37,6 +38,11 @@ func (r *Repository) Find(filters map[string]interface{}) ([]*domain.Resource, e
 	if conditions.IDs != nil {
 		db = db.Where(conditions.IDs)
 	}
+
+	if filters["isDeleted"] != nil {
```

Comment:
@rahmatrhd Added this to first validate if a `isDeleted ` filter is being used or not. Using conditions.IsDeleted would be misleading as it would be set to false when the filter is not being applied.